### PR TITLE
changes to channel management and clightning_bitcoin_spark

### PR DESCRIPTION
**Pull request by eoliveros**
- [x] wrapped the channel management page to be the same as the other pages.
- [x] moved the buttons to the navbar
  - [x] removed the old buttons from the ln_wallet blueprint
  - [x] removed the old buttons on the reporting blueprint
  - [x] created new buttons on the navbar. only shows certain buttons on specific blueprint.
- [x] close button in channel management not available when channel status is not normal 
- [x] remove clightning_bitcoin_spark
  - [x] commented lines in the docker-compose.yml file for clightning_bitcoin_spark
  - [x] commented the nginx lines for clightning_bitcoin_spark
  - [x] open/close channel to make sure its working after commenting the clightning_bitcoin_spark lines.
  - [x] remove the lines in the config for clightning_bitcoin_spark in nginx and docker-compose,yml
- [x] change the way the channels are displayed.
  - [x] putting the block as a collapsable block

### CODE REVIEW

File: templates/lightning/channel_management.html [hunk 1]
```diff
@@ -58,17 +61,20 @@ <h2>Channels</h2>
         <li><strong>Age:</strong> n/a</li>
         <li><strong>Peer:</strong> <small class="break-all">{{ channel.peer.id }}</small></li>
         <li>
+        {% if channel.peer.connected %}
```
Comment (djpnewton) [inline templates/lightning/channel_management.html:64]: we might still want to close the channel even if the peer is not connected.

File: templates/layout.html [hunk 2]
```diff
@@ -23,11 +23,15 @@
...
-          <li class="nav-item active">
-    	    <a class="nav-link" href="/">Home<span class="sr-only">(current)</span></a>
-          </li>
-          -->
+          {% if request.blueprint == "ln_wallet" %}
+            {% if request.endpoint != "ln_wallet.ln_index" %}
+	      <a class={% if request.endpoint == "ln_wallet.ln_index" %}"btn btn-primary"{% else %}"btn btn-secondary"{% endif %}href="/ln_wallet"><i class="glyphicon glyphicon-chevron-left"></i>Lightning Home</a>&nbsp
+            {% endif %}
+          {% elif request.blueprint == "reporting" %}
+	  <a class={% if request.endpoint == "reporting.dashboard_general" %}"btn btn-primary" {% else %}"btn btn-secondary"{% endif %}href="dashboard_general"><i class="glyphicon glyphicon-chevron-left"></i>General</a>&nbsp
```
Comment (djpnewton) [inline templates/layout.html:31]: fix spacing

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: templates/layout.html (hunk 2 above)
Comment (eoliveros) [reply templates/layout.html]: fixed the spacing.

File: templates/lightning/channel_management.html (hunk 1 above)
Comment (eoliveros) [reply templates/lightning/channel_management.html]: using channel.state instead of channel.peer.connected.

**Pull request merged**